### PR TITLE
Upgrade node version in dockerfile

### LINKED ISSUE #352: [refactor] upgrade node version in Dockerfile
file: `Dockerfile`

current Node version: `node:12.22`
Alternative upgrades: ~`node:16.4.0`~ `node:16.3.0`

### PR DESCRIPTION
**Pull request by RhnSharma**
Fixes #352
Hi @BlackHole1, how does this look?
Let me know if this needs any changes.
Thanks

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:12.22 as base
+FROM node:16.4.0 as base
```
Comment (BlackHole1) [inline Dockerfile:1]:
```suggestion
FROM node:16.3.0 as base
```

File: Dockerfile [hunk 2]
```diff
@@ -22,7 +22,7 @@ RUN yarn install --production --frozen-lockfile
 
 # ----
 
-FROM node:12.22
+FROM node:16.4.0
```
Comment (BlackHole1) [inline Dockerfile:25]:
```suggestion
FROM node:16.3.0
```

**Review by BlackHole1 — CHANGES_REQUESTED** [review]
`node:16.4.0` does not exist for the time being

link: [https://hub.docker.com/_/node?tab=tags&page=1&ordering=last_updated&name=16.](https://hub.docker.com/_/node?tab=tags&page=1&ordering=last_updated&name=16.)

**Comment by RhnSharma** [comment]
@BlackHole1 I have made the changes you suggested. Please take a look at it and let me know if this needs any changes.
Thanks

**Review by BlackHole1 — APPROVED** [review]

**Comment by BlackHole1** [comment]
Thank you for your contribution, we will contact you recently to send our gift

**Pull request merged**